### PR TITLE
Validate contact form

### PR DESCRIPTION
**Pull request by Krishnabot**
In this pull request
1. When the user submits the form, check if the email is in lower case.
2. If the validation is OK, the form is sent.
3. If the validation is not OK,  shows an error message to the user near the submit button informing them of the error, and the form is not sent.

### CODE REVIEW

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Krishnabot , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has contributions from one individual, and it's a group project, kindly ensure everyone contributes to the project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: script.js [hunk 1]
```diff
@@ -25,3 +25,12 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
   navBar.classList.remove('active');
   navLink.classList.remove('active');
 }));
+/* ------------Section for Pop up--------------  */
+
+document.querySelector('.submit').addEventListener('click', (event) => {
+  const email = document.querySelector('.email');
+  if (email.checkValidity(/[A-Z]/.test(email))) {
+    event.preventDefault();
+    email.setCustomValidity('Email must be lowercase. Please Try agin');
```
Comment (tufoinnkuo10) [inline script.js]: - [ ] Great job.. your validation works well. one little issue, kindly make the error message to show beside the submit button. 
- [ optional] you can make the font color red so the warning is noticed

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Krishnabot 

Great job so far for doing the good work on your own. nevertheless, there are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅email checks for lowercase well
✅ error message shows

# Required Changes ♻️
Check the comments under the review

tufoin

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: script.js [hunk 2]
```diff
@@ -25,3 +25,15 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
...
+/* ------------Section for Pop up--------------  */
+
+const form = document.querySelector('.contact-form');
+const email = document.querySelector('.email');
+form.addEventListener('submit', (event) => {
+  if (email.checkValidity(/[A-Z]/.test(email))) {
+    event.preventDefault();
+    document
+      .querySelector('.submit')
+      .setCustomValidity('Email must be lowercase. Please Try agin');
```
Comment (tufoinnkuo10) [inline script.js]: - [ ] validation works correctly, the error message is in the right position but the form doesn't submit even when the email is in lowercase. rather it keeps giving the error message when I try to submit. 
Screenshot (155)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Krishnabot 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ form works well


Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by roshan-bajgain — APPROVED** [review]
Hi @Krishnabot, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**